### ISSUE TITLE
UnboundLocalError: cannot access local variable 'motor_volt' where it is not associated with a value

### ISSUE THREAD

**Issue by scottgrobinson** [itchannel/fordpass-ha#418]
Hi!

Version 1.60

I have an automation that calls "Turn on fordpass_ignition_Switch" which worked the first time I called it, but appears to now be stuck and won't re-run caused by the following error:

```
2023-10-30 09:29:47.349 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 233, in _handle_refresh_interval
    await self._async_refresh(log_failures=True, scheduled=True)
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 389, in _async_refresh
    self.async_update_listeners()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 172, in async_update_listeners
    update_callback()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 469, in _handle_coordinator_update
    self.async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 779, in async_write_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 879, in _async_write_ha_state
    state, attr = self._async_generate_attributes()
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 823, in _async_generate_attributes
    attr.update(self.extra_state_attributes or {})
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/fordpass/sensor.py", line 450, in extra_state_attributes
    return self.get_value("attribute")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/fordpass/sensor.py", line 260, in get_value
    if motor_volt != 0 and motor_amps != 0:
       ^^^^^^^^^^
UnboundLocalError: cannot access local variable 'motor_volt' where it is not associated with a value
```

Vehicle JSON if required @ https://pastebin.com/fCYY6sZD (Expires in 1 month from now)

**Comment by scottgrobinson** [comment]
Infact - I appologise, this appears to have been resolved in an upcoming beta. I will test that out but please close this if that's the case. Thank you!

**Comment by SquidBytes** [comment]
Yeah that's a known bug I inadvertently added in 1.60 Release.
It is fixed in 1.60 Beta's, and will be fixed in 1.61 Release.
